### DISCUSSION TITLE
在部署到vercel时出现了问题，有人帮我解答一下吗？

### DISCUSSION THREAD

**Discussion by Bashir-1** [qier222/YesPlayMusic#2015] [Q&A] [answered]
![网页捕获_27-3-2023_14613_vercel.com](https://i2.100024.xyz/2023/03/27/n9cm4v.webp)
**不知道出现什么问题了**

**Comment by tosasitill** [comment]
换一下版本到16.X！
https://github.com/qier222/YesPlayMusic/issues/1860

**Reply by Bashir-1** [reply]
![image](https://user-images.githubusercontent.com/120711536/228141539-a9a383bb-56c8-4f7a-b79c-6049c3cd35b8.png)
重新又搞了一个，但又提示出现错误，不知道怎么弄😭😭

**Reply by tosasitill** [reply, marked as answer]
我也这样 我新开个 issue 问问
![D12716F2-7E66-42FA-A121-61CA094C4A9A](https://user-images.githubusercontent.com/66861785/228233139-a9bcfd34-5b2d-4335-bfb3-5f7f99b7a2ee.png)
